lib/mozart_fetcher.ex

```diff
@@ -25,7 +25,7 @@ defmodule MozartFetcher do
     case environment() do
       :prod -> [certfile: cert_path(), keyfile: key_file_path()]
       _ -> [certfile: System.get_env("DEV_CERT_PEM")]
-    end
+    end ++ [verify: :verify_none]
```

Comment:
Not sure if it would read better, but do you think we could pattern match on the environment and add `verify none` to both?

Reply:
Ill just add it to both lists, don't think we can pattern match as environment is a function :)